### PR TITLE
Use cp instead of rsync to copy directories

### PR DESCRIPTION
**Pull request by gjuric**
Since rsync is not installed on every host it's better to use standard tools.
Previosly rsync was introduced ( #1072 ) to enable copying of subdirectories, but
this is doable with cp as well.

- [ ] Bug fix #…?
- [x] New feature #2656
- [ ] BC breaks?
- [ ] Deprecations?
- [ ] Tests added?
- [x] Changelog updated?

### CODE REVIEW

File: recipe/deploy/copy_dirs.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
         foreach (get('copy_dirs') as $dir) {
             if (test("[ -d {{previous_release}}/$dir ]")) {
                 run("mkdir -p {{release_path}}/$dir");
-                run("rsync -av {{previous_release}}/$dir/ {{release_path}}/$dir");
```
Comment (antonmedv) [inline recipe/deploy/copy_dirs.php:10]: Should we check what `$dir` does not contain `/` at the end? 

Will be nice to throw an exception with explanation of why to remove `/` from copy_dirs.

Comment (gjuric) [reply recipe/deploy/copy_dirs.php:10]: I can also just trim `/` from the provided strings, in that case we don't need to throw an exception, what do you think?

Comment (antonmedv) [reply recipe/deploy/copy_dirs.php:10]: I think better to throw exception with explanation. In future it will help.

Comment (gjuric) [reply recipe/deploy/copy_dirs.php:10]: Done.

**Comment by gjuric** [comment]
I had to push a fix that strips the last name of the folder in the destination path for things to work properly.

**Comment by antonmedv** [comment]
Please check errors.

**Comment by gjuric** [comment]
It looks like an error that is not in the code I wrote, but it was fixed in https://github.com/deployphp/deployer/pull/2668

File: recipe/deploy/copy_dirs.php [hunk 2]
```diff
@@ -5,9 +5,16 @@
...
+                throw new \RuntimeException('Entries in config parameter "copy_dirs" must not end with "/"');
+            }
             if (test("[ -d {{previous_release}}/$dir ]")) {
+                $destinationDir = '';
+                if (strpos($dir, '/') !== false) {
+                    $destinationDir = substr($dir, 0, strrpos($dir, '/') + 1);
+                }
                 run("mkdir -p {{release_path}}/$dir");
-                run("rsync -av {{previous_release}}/$dir/ {{release_path}}/$dir");
+                run("cp -rpf {{previous_release}}/$dir {{release_path}}/$destinationDir");
```
Comment (Bilge) [inline recipe/deploy/copy_dirs.php:17]: This should probably be `cp -af`.

**Pull request merged**